src/ert/run_arg.py

```diff
@@ -29,8 +29,10 @@ def create_run_arguments(
     runpaths: Runpaths,
     active_realizations: Union[List[bool], npt.NDArray[np.bool_]],
     ensemble: Ensemble,
+    iteration: Optional[int] = None,
 ) -> List[RunArg]:
-    iteration = ensemble.iteration
+    if iteration is None:
+        iteration = ensemble.iteration
```

Comment:
This is an interesting question. Now that I think about it. It could be abused to create a `runpath` containing an iteration that does not match the `ensemble` iteration. I will try removing this change and working around it.

Reply:
Ok give it a try and if I remove this I will need to add another helper function to create a `List[RunArg]` that will be used to load from data from a forward model custom iteration. This new function could also be used to make a runpath where the ensemble iteration does not match the runpath iteration. So I am unsure how much this will help prevent future headache if somebody really wants to have an ensemble iteration out of sync with the runpath iteration. 

As far as I can tell this can also be achieved now if somebody in the code decides to update the ensemble iteration number directly. They will be out of sync with the runpath iteration and there is no hard check for this as far as I could find.